### DISCUSSION TITLE
More than 9 rows in MSSQL wont display data

### DISCUSSION THREAD

**Discussion by piperpipuk** [Budibase/budibase#3720] [Help]
Hi, After the new update, if I have more than 9 rows in my table, Budibase won't display the records.

![image](https://user-images.githubusercontent.com/11796003/145226941-e60c9bce-3056-40e4-94ec-1dab5e704fbe.png)

![image](https://user-images.githubusercontent.com/11796003/145227011-7c983a42-76a6-47dd-9819-b200e6825c0d.png)

Less than 10 rows works as expected

![image](https://user-images.githubusercontent.com/11796003/145227155-0300bf4e-ff6f-41a4-831d-c03d4ed92220.png)

![image](https://user-images.githubusercontent.com/11796003/145227195-554553c3-3274-4343-9e78-20425c05c002.png)

**Comment by mjashanks** [comment]
Hi @UKXL 

I tried to reproduce this and couldn't. My Steps...
1. Created a new table
2. Added 15 rows
3. Added MySQL datasource
3. In `Design`, added screens for my new table
4. On  the list screen, I set the `Data Provider`'s `Limit` to `50`

All 15 rows are displayed.

If you send me a `CREATE` script for your table (no need for the data), I will try and reproduce it with that.

Thanks.

**Reply by piperpipuk** [reply]
Hi, Sorry for the delay,

```
CREATE TABLE Sample (
  id              INT           NOT NULL    IDENTITY    PRIMARY KEY,
  name           VARCHAR(100)  NOT NULL,
);
```
